<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KQUI_MainTitleFrame</class>
 <widget class="QWidget" name="KQUI_MainTitleFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>20</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="buttonContainer">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>0</y>
     <width>300</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="KQPushButton" name="pbCheckTimer">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>0</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>時</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="KQPushButton" name="pbMinimize">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>0</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="KQComboBox" name="comboBoxZoom">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>0</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>等倍</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>50%</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>75%</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>100%</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>125%</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>150%</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>175%</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>200%</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>等倍</string>
     </property>
    </item>
   </widget>
   <widget class="KQPushButton" name="pbTrasparent">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>0</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>透</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="KQPushButton" name="pbCheckInfo">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>0</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>情</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="KQPushButton" name="pbClose">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>0</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="KQPushButton" name="pbMinimizeSelf">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>0</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KQPushButton</class>
   <extends>QPushButton</extends>
   <header>kqpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KQComboBox</class>
   <extends>QComboBox</extends>
   <header>kqcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
